src/others/components/FilterDropdown/ListWithSearch.tsx

```diff
@@ -0,0 +1,113 @@
+import { FixedSizeList, areEqual } from "react-window";
+import { Button, ListItem, ListItemButton, ListItemText, ClickAwayListener, Divider, useAutocomplete, Typography } from "@mui/material";
+import { FunctionComponent, memo, forwardRef } from "react";
+import SearchIcon from "@mui/icons-material/Search";
+
+import { FilterItem } from "../../contexts/filter";
+
+import styles from "./ListWithSearch.module.css";
```

Comment:
**issue**

Please use `styled-components` for consistency.

Reply:
I went this route, as I noticed a handful of other files like this. Happy to change if `styled-components` is the preferred approach.

Reply:
I might be missing something, but I don't see `styled-components` installed as a dependency. Can you confirm? Perhaps this can also be addressed in a follow-up PR?

Reply:
I was thinking of the syntax, sorry. 
We have `@emotion/styled` installed.